bayesopt-workflow/cifar.py

```diff
@@ -0,0 +1,142 @@
+'''Train a simple deep CNN on the CIFAR10 small images dataset.
```

Comment:
was this code taken from somewhere else? If so, we should probably put an attribution if we are going to put it on our public repo